src/instructlab/config/show.py

```diff
@@ -1,10 +1,17 @@
 # SPDX-License-Identifier: Apache-2.0
+# Standard
+import sys
+
 # Third Party
 import click
-import yaml
+import ruamel.yaml
```

Comment:
Shouldn't ruamel.yaml be in requirements.txt?

Reply:
`pydantic_yaml` has `ruamel.yaml` as a dependancy. But I'll add it. Thanks.